R/model_profile.R

```diff
@@ -54,7 +56,7 @@ model_profile <- function(explainer,
                           ...,
                           groups = NULL,
                           k = NULL,
-                          center = TRUE,
+                          center = FALSE,
```

Comment:
"center" argument seems to not function properly for both ALE and PDP+ICE plots

-> ALE: from my understanding of the surv_ale function, "center=TRUE" should give the classic ALE values in units as differences from to mean prediction, but outcome values do not change compared to "center = FALSE", (the surv_ale function however, calculates both values correctly, I assume that user-specified argument is not passed on properly to the surv_ale function

-> PDP(+ICE): it is not quite clear to me, what the argument does here. From what I can see, it is passed on to the surv_aggregate_profiles function, but this function never actually uses it. I think it would be nice to implement centering as described by Goldstein & Kapelner (2015), as it is very helpful to properly interpret PDP and ICE plots

Reply:
thanks! should be OK in the new version -- `center` works for PDP, ALE and ICE